templates/nextjs/package.json

```diff
@@ -17,15 +17,12 @@
     "@fuels/react": "0.36.1",
     "@tanstack/react-query": "5.55.4",
     "clsx": "2.1.1",
-    "@wagmi/connectors": "5.1.14",
```

Comment:
I've tested out the [release version](https://www.npmjs.com/package/fuels/v/0.0.0-pr-3554-20250106201303) on `create fuels` and it seems these are no longer needed for `fuels/connectors` although I couldn't find a corresponding fix in the issues on that repo.

Reply:
@maschad Should we also remove the `@wagmi/core`?

Reply:
@Torres-ssf This is getting merged; if you think this is a thing, please open another PR.